### PR TITLE
ui: show external access warning

### PR DESCRIPTION
**Pull request by mynameisbogdan**

### CODE REVIEW

File: src/Jackett.Common/Content/index.html [hunk 1]
```diff
@@ -50,6 +50,10 @@
         </div>
         <hr />
 
+        <div id="warning-external-access" hidden class="alert alert-danger text-center" role="alert">
+            <strong><span class="glyphicon glyphicon-alert"></span> Your instance has external access enabled without using an admin password <span class="glyphicon glyphicon-alert"></span></strong>
+        </div>
```
Comment (garfield69) [inline src/Jackett.Common/Content/index.html:55]: perhaps add why this is not a good idea, i.e. this is a security risk.

File: src/Jackett.Common/Content/index.html [hunk 2]
```diff
@@ -50,6 +50,10 @@
         </div>
         <hr />
 
+        <div id="warning-external-access" hidden class="alert alert-danger text-center" role="alert">
+            <strong><span class="glyphicon glyphicon-alert"></span> Your instance has external access enabled without using an admin password <span class="glyphicon glyphicon-alert"></span></strong>
```
Comment (garfield69) [inline src/Jackett.Common/Content/index.html:54]:
```suggestion
            <strong><span class="glyphicon glyphicon-alert"></span>Security Risk: Your instance has external access enabled without using an admin password.<span class="glyphicon glyphicon-alert"></span></strong>
```

**Review by garfield69 — CHANGES_REQUESTED** [review]

**Review by garfield69 — APPROVED** [review]

**Comment by garfield69** [comment]
Tested. lGtM

**Comment by virusperfect** [comment]
Can this be disabled? It's super annoying if I already have an authorization layer on top of Jackett.

**Comment by ilike2burnthing** [comment]
Add the admin password, sign in once, you won't be asked again unless you delete cookies or use a different browser/device.

**Pull request merged**